apps/admin-gui-e2e/src/integration/facility-management-perun_admin.spec.js

```diff
@@ -0,0 +1,212 @@
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=resources]')
+      .click()
+      .get(`[data-cy=${dbResourceName}-checkbox]`)
+      .click()
+      .get('[data-cy=delete-resource-button]')
+      .click()
+      .get('[data-cy=delete-button-dialog]')
+      .click()
+      .wait('@deleteResource')
+      .wait('@getAssignedResources')
+      .get(`[data-cy=${dbResourceName}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=attributes]')
+      .click()
+      .get('[data-cy=add-attributes]')
+      .click()
+      .get('[data-cy=filter-attributes]')
+      .type('Login namespace')
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .type('einfra')
+      .get('[data-cy=save-selected-attributes]')
+      .click()
+      .wait('@setAttributes')
+      // assert that attribute exists
+      .get(`[data-cy=${addedAttribute}-value]`)
+      .should('exist')
+  });
+
+  it('test delete attribute', () => {
+
+    cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+      .intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=attributes]')
+      .click()
+      .get(`[data-cy=${deleteAttribute}-checkbox]`)
+      .click()
+      .get('[data-cy=remove-attributes]')
+      .click()
+      .get('[data-cy=delete-attributes]')
+      .click()
+      .wait('@removeAttributes')
+      // assert that attribute exists
+      .get(`[data-cy=${deleteAttribute}-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add facility manager', () => {
+
+    cy.intercept('**/authzResolver/setRole/**').as('setRole')
+      .intercept('**/facilitiesManager/getEnrichedFacilities').as('getEnrichedFacilities')
+      .intercept('**/usersManager/findRichUsersWithAttributes?**').as('findRichUsers')
+      .wait('@getEnrichedFacilities')
+      .get(`[data-cy=${dbFacilityName2}]`)
+      .click()
+      .get('[data-cy=advanced-settings]')
+      .click()
+      .get('[data-cy=managers]')
+      .click()
+      .get('[data-cy=add-manager-button]')
+      .click()
+      .get('[data-cy=search-manager-input]')
+      .type(`${addManagerUser}`)
+      .get('[data-cy=search-manager-button]')
+      .click()
+      .wait('@findRichUsers')
+      .get(`[data-cy=${addManagerUser}-checkbox]`)
+      .click()
+      .get('[data-cy=add-manager-button-dialog]')
+      .click()
+      .wait('@setRole')
+      // assert that manager was added
+      .get(`[data-cy=${addManagerUser}-checkbox]`)
+      .should('exist')
+  });
```

Comment:
The same case occurs in this dialog (but for me not every time I tried to run e2e tests). If you want you can fix also this behavior.
![Screenshot from 2022-03-01 21-21-59](https://user-images.githubusercontent.com/70371638/156246511-f7fb0049-debb-435d-acf5-2716b743a12a.png)